discord/sku.go

```diff
@@ -0,0 +1,36 @@
+package discord
+
+import (
+	"time"
+
+	"github.com/disgoorg/snowflake/v2"
+)
+
+type SKU struct {
```

Comment:
I left out the `manifest_labels` field as it's undocumented and its type is not clear: https://github.com/discord/discord-api-docs/blob/main/docs/monetization/SKUs.md?plain=1#L25